_src/static/sass/main-ltr.scss

```diff
@@ -13,17 +13,21 @@
 @import '../../vendor/bi-app/bi-app-ltr';   // LTR support
 
 // ------------------------------
-// #UTILITIES
```

Comment:
This compile structure and some of the file/folder organization does not match our planned for compile order in https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#main-style-compile. Please sync up.